docs/appendix/return-codes.md

```diff
@@ -0,0 +1,19 @@
+# Return Codes
+
+Return codes, sometimes called exit codes, are numbers returned from the `rig` executable 
+to indicate the status of the command when on exit.  Exit codes can be interpreted by 
+machine scripts to adapt in the event of successes of failures. Generally a return code 
+of `0` is used to indicate success.  Non-zero return codes are used to communicate that
+and error has occurred and can give an idea of what type of error was raised.
+
+`rig` exit codes used to indiciate certain conditions and are as follows:
+
+* `1` - Unexpected
+  * Returned when an uncaught error is raised and the program exited unexpectedly
```

Comment:
this is also used by the urfave/cli framework when an error is caught in it's own code, and sometimes is returned by rig project script.

Reply:
Maybe we should just call 1 the default.